route/route.go

```diff
@@ -631,16 +529,6 @@ func makeDecoders(num int) (chan *zstd.Decoder, error) {
 func unmarshal(r *http.Request, data io.Reader, v interface{}) error {
 	switch r.Header.Get("Content-Type") {
 	case "application/x-msgpack", "application/msgpack":
-		// First try to decode with the fast but fussy msgp. If that fails,
-		// defer to the much more friendly msgpack.
-		if decodable, ok := v.(msgp.Decodable); ok {
-			err := msgp.Decode(data, decodable)
-
-			if err == nil {
-				return nil
-			}
-		}
-
```

Comment:
why do we not want to do this?

Reply:
We never did do it. This codebase contains no msgp.Decodable data structures. Looks like it was just cargo-culted over from shepherd.